submission_runner.py

```diff
@@ -282,18 +283,26 @@ def train_once(workload: spec.Workload,
     metrics_logger = logger_utils.set_up_loggers(log_dir, flags.FLAGS)
     workload.attach_metrics_logger(metrics_logger)
 
+  global_start_time = time.time()
   if USE_PYTORCH_DDP:
     # Make sure all processes start training at the same time.
-    dist.barrier()
+    global_start_time_tensor = torch.tensor(
```

Comment:
can you make this a helper function in [`pytorch_utils.py`](https://github.com/mlcommons/algorithmic-efficiency/blob/main/algorithmic_efficiency/pytorch_utils.py) that returns a synced time if using DDP (and then call it below where we need synced times too)? but please keep the comment here saying we need to sync the times